web/src/routes/(user)/albums/[albumId]/+page.svelte

```diff
@@ -384,176 +433,225 @@
-                  forceDark
-                  size="20"
-                  icon={mdiPlus}
-                  on:click={() => (viewMode = ViewMode.SELECT_USERS)}
-                  title="Add more users"
-                />
               {/if}
-            </div>
+            </section>
           {/if}
 
-          <!-- ALBUM DESCRIPTION -->
-          {#if isOwned || album.description}
-            <button
-              class="mb-12 mt-6 w-full border-b-2 border-transparent pb-2 text-left text-lg font-medium transition-colors hover:border-b-2 dark:text-gray-300"
-              on:click={() => (isEditingDescription = true)}
-              class:hover:border-gray-400={isOwned}
-              disabled={!isOwned}
-              title="Edit description"
-            >
-              {album.description || 'Add description'}
-            </button>
+          {#if album.assetCount === 0}
+            <section id="empty-album" class=" mt-[200px] flex place-content-center place-items-center">
+              <div class="w-[300px]">
+                <p class="text-xs dark:text-immich-dark-fg">ADD PHOTOS</p>
+                <button
+                  on:click={() => (viewMode = ViewMode.SELECT_ASSETS)}
+                  class="mt-5 flex w-full place-items-center gap-6 rounded-md border bg-immich-bg px-8 py-8 text-immich-fg transition-all hover:bg-gray-100 hover:text-immich-primary dark:border-none dark:bg-immich-dark-gray dark:text-immich-dark-fg dark:hover:text-immich-dark-primary"
+                >
+                  <span class="text-text-immich-primary dark:text-immich-dark-primary"
+                    ><Icon path={mdiPlus} size="24" />
+                  </span>
+                  <span class="text-lg">Select photos</span>
+                </button>
+              </div>
+            </section>
           {/if}
-        </section>
+        </AssetGrid>
       {/if}
-
-      {#if album.assetCount === 0}
-        <section id="empty-album" class=" mt-[200px] flex place-content-center place-items-center">
-          <div class="w-[300px]">
-            <p class="text-xs dark:text-immich-dark-fg">ADD PHOTOS</p>
-            <button
-              on:click={() => (viewMode = ViewMode.SELECT_ASSETS)}
-              class="mt-5 flex w-full place-items-center gap-6 rounded-md border bg-immich-bg px-8 py-8 text-immich-fg transition-all hover:bg-gray-100 hover:text-immich-primary dark:border-none dark:bg-immich-dark-gray dark:text-immich-dark-fg dark:hover:text-immich-dark-primary"
-            >
-              <span class="text-text-immich-primary dark:text-immich-dark-primary"
-                ><Icon path={mdiPlus} size="24" />
-              </span>
-              <span class="text-lg">Select photos</span>
-            </button>
-          </div>
-        </section>
+      {#if album.sharedUsers.length > 0 && !$showAssetViewer}
+        <div class="z-[9999] absolute bottom-0 right-0 mb-6 mr-6 justify-self-end">
```

Comment:
Is that z-index avoidable? I hate it when we do that and I would really like to (slowly) get away from this in all places.

Reply:
Yes this one is.